initial hgvs-convert branch publish

AkRampersad opened the pull request:
upload a MVP hgvs-converter to be used as a common module.

Looking to implement in the future without web call.

Entrez email can change if desired.

AkRampersad committed:
initial hgvs-convert branch publish

upload a MVP hgvs-converter to be used as a common module.

Looking to implement in the future without web call.

Entrez email can change if desired.

rkimoakbioinformatics commented:
I tried `var_example` in the code. The variant is in HGVS c. notation, where NM_001366781 is a transcript. `obtain_chromosome` got the correct chromosome. However, the position was wrong. It gave 90 which was cDNA position, but we need the conversion from the cDNA position 90 to a genomic position on chr1. Is there a way to get the g. position? Maybe Entrez.efetch?

AkRampersad committed:
cHGVS to gHGVS mapping 

added mapping for cHGVS to gHGVS variants with a caveat that if there is no sequence alignments available for the transcript from the Universal Transcript Archive (UTA), an error will be thrown. 

A new dataprovider can be written/provided by subclassing hgvs.dataproviders.interface which is a further improvement.

AkRampersad commented:
I do not believe that Entrez is able to convert from cHGVS to gHGVS accurately. While it can convert from cDNA to genomic pos Entrez does not account for introns, UTRs, or alternative splicing. I took this as being inaccurate and tried to implement the conversion using hgvs c_to_g mapping method which does take into account introns, UTRS, and alt splicing . If this is a wrong assumption please let me know and I will gladly change it to utilize Entrez.efetch.
